launch

```diff
@@ -141,6 +145,8 @@ if __name__ == '__main__':
                               (', '.join(sim_choices.keys()), sim_default)))
     parser.add_argument('-t', '--test', action='store_true',
                         help='Use current stdin & stdout for simulated UART.')
+    parser.add_argument('--ramdisk',
+                        help='Give initial ramdisk to OVPSim. '),
```

Comment:
Right! Otherwise we cannot run tests on Travis, that require ramdisk.

@coodie Please make sure ramdisk works with QEMU as well!